optimizer/src/main/java/com/netease/arctic/optimizer/local/LocalOptimizer.java

```diff
@@ -150,6 +151,10 @@ public void stop() throws IOException {
 
   public static void main(String[] args) throws CmdLineException {
     OptimizerConfig optimizerConfig = new OptimizerConfig(args);
+    if (optimizerConfig.getOptimizerId() == null || optimizerConfig.getOptimizerId().isEmpty() ||
```

Comment:
I think not only external optimizer but also "internal" optimizer can `register` to ams, and then the start script of external/internal optimizer could be unified.

Reply:
It confusing that the start script of internal optimizer use `--queue-id`, but external optimizer use `--queue-name`

Reply:
> It confusing that the start script of internal optimizer use `--queue-id`, but external optimizer use `--queue-name`

the start script of `internal` optimize is generate with ams , so the `queue-id` is easy to get. but external optimizer is started by user who get `queue-id` diffcult

Reply:
what if user starts external optimizer but set optimizerId?

Reply:
> what if user starts external optimizer but set optimizerId?

it also work well.